cypress/support/Helpers/apiHelper.ts

```diff
@@ -1,28 +1,54 @@
+import CandidatePayloadInit from "../Initializers/CandidatePayloadInit";
 import UserPayLoadInit from "../Initializers/UserPayloadInit"
-export enum StatusCode{
- Success = 200,
+export enum StatusCode {
+    Success = 200,
 }
-export  const Urls ={
-    employees: `${Cypress.env("apiLogin")}/employees`
+export const Urls = {
+    employees: '/web/index.php/api/v2/employees',
+    viewCandidates: '/web/index.php/api/v2/recruitment/candidates?limit=50&offset=0&model=list&sortField=candidate.dateOfApplication&sortOrder=DESC',
+    addCandidate: '/web/index.php/api/v2/recruitment/candidates',
+    shortlistCandidate: '/web/index.php/api/v2/recruitment/candidates/',
+    viewCandidateProfile: '/web/index.php/recruitment/addCandidate/'
 }
-export class apiHelper{
-    static addNewEmployee() {
-    return cy.fixture("employeeInfo").then(employeeData => {
-        cy.request({
+export class apiHelper {
+    static addNewEmployeeViaAPI() {
+        return cy.request({
             method: 'Post',
-            url: `${Cypress.env("apiLogin")}/employees`,
+            url: Urls.employees,
+            body: UserPayLoadInit.inituser()
+        })
+    }
+
+    static getTotalCandidatesRecordsViaAPI() {
+        return cy.intercept(Urls.viewCandidates).as('interception').wait('@interception').then((interception) => {
+            return interception.response.body.meta.total
+        })
+    }
+
+    static addCandidateViaAPI() {
+        return cy.request({
+            method: 'Post',
+            url: Urls.addCandidate,
+            body: CandidatePayloadInit.initCandidate()
+        }).then((response) => {
+            return response.body.data.id
+        })
+    }
+
+    static shortListCandidateViaAPI(id: String) {
+        return cy.request({
+            method: 'PUT',
+            url: `${Urls.shortlistCandidate}${id}${"/shortlist"}`,
             body: {
-                firstName: employeeData.firstName,
-                middleName: employeeData.middleName,
-                lastName: employeeData.lastName,
-                empPicture: employeeData.empPicture,
-                employeeId: employeeData.empId
+                "note": null
             }
-
         })
-    })
-}
-static addNewEmployeeViaAPI() {
-    return cy.addNewEmployee(Urls.employees, UserPayLoadInit.inituser());
-}
+    }
+
+    static viewCandidatePageViaAPI(id: String) {
```

Comment:
`openEditCansdidatePage(id: string)`
and it should have intercepts and waiting for the intercepted APIs to finish, that would make your test more stable